### DISCUSSION TITLE
Why do I still get an error when executing the following load under the latest version of fio?

### DISCUSSION THREAD

**Discussion by clf-123** [axboe/fio#1842] [Q&A] [answered]
[global]
name=bw
filename=/dev/nvme0n1
ioengine=io_uring
direct=1
bs=8K 
group_reporting
zonemode=zbd
thread=1
lat_percentiles=1
iodepth=1
zonesize=64M
time_based
runtime=1m 

[write]
ramp_time=5s
numjobs=4
rw=write 
bs=8k
exitall

[finish]
new_group
numjobs=1
rw=write
finish=1
bs=8K
rate=7M
exitall

It prompts me:Bad option <finish=1>

**Comment by sitsofe** [comment, marked as answer]
@clf-123 :
`finish` isn't an fio option. In the job file you posted you have this:

```
[...]
[finish]
new_group
numjobs=1
rw=write
finish=1 <---
bs=8K
[...]
```

Why do you have that line? `finish` is not an option listed on https://fio.readthedocs.io/en/latest/fio_doc.html ...

**Comment by sitsofe** [comment]
Closing due to lack of reply from reporter. If this issue is still happening with the latest fio (see https://github.com/axboe/fio/releases to find out which version that is) please reopen. Thanks!
